docs/repositories/commits.md

```diff
@@ -125,7 +125,12 @@ The option **Show files without code quality changes** allows you to list all fi
 
 ## Diff tab
 
-The **Diff** tab displays the line differences in each file that was changed in the {{ page.meta.page_name }}.
+The **Diff** tab displays the differences in each file that was changed in the {{ page.meta.page_name }}. The background of the lines that are different depend on the change:
+
+-   **Red**: Deleted line
+-   **Yellow**: Original version of a changed line with the updated characters highlighted in red
+-   **Pale green**: New version of a changed line with the updated characters highlighted in bright green
```

Comment:
If this behaves anything like the GitHub version (to me looks like it does after double-checking), I'd tweak as follows for clarity:

```suggestion
-   **Yellow**: Old version of a changed line with deleted characters highlighted in red
-   **Pale green**: New version of a changed line with added characters highlighted in bright green
```